src/libs/ReportActionsUtils.js

```diff
@@ -144,7 +144,12 @@ function getLastVisibleAction(reportID, actionsToMerge = {}) {
  */
 function getLastVisibleMessageText(reportID, actionsToMerge = {}) {
     const lastVisibleAction = getLastVisibleAction(reportID, actionsToMerge);
-    const message = lodashGet(lastVisibleAction, ['message', 0]);
+    const message = lodashGet(lastVisibleAction, ['message', 0], {});
+
+    if (_.isEmpty(message)) {
+        return '';
+    }
+
```

Comment:
```suggestion
```

Not required

Reply:
I think you might have missed my comment https://github.com/Expensify/App/issues/17164#issuecomment-1506972128.

> For your 2nd optimization, we should keep the check for falsy message. Returning an empty object will not be enough since the [isReportMessageAttachment()](https://github.com/Expensify/App/blob/feedecf0c6d218bff94abbf4943b9a69bbad374a/src/libs/isReportMessageAttachment.js#L10) function, which use the falsy message, is assuming the message would have text and html field. I don't want to modify this function, so the falsy message check should be kept.

Reply:
I didn't miss your comment :sweat_smile:. There is no problem in passing an empty object to [isReportMessageAttachment](https://github.com/Expensify/App/blob/feedecf0c6d218bff94abbf4943b9a69bbad374a/src/libs/isReportMessageAttachment.js#L10).

Reply:
Sorry for the confusion. You are right, after adding the default `{}` value, we wouldn't need this guard condition anymore.